### ISSUE TITLE
Document `scope` in the context of configuration comments

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#8077]
### Describe the documentation issue

see: https://stylelint.io/user-guide/cli#--report-invalid-scope-disables---risd

> `--report-invalid-scope-disables, --risd`
> Produce a report of the `stylelint-disable` comments that used for rules that don't exist within the configuration object. More info.

This comment doesn't explain what `scope` means and leaves it up to the reader to deduce that it means "a comment controlling behavior for a specific rule".

----

see: https://stylelint.io/user-guide/ignore-code

> Or you can turn off individual rules:
> 
> ```css
> /* stylelint-disable selector-max-id, declaration-no-important */
> #id {
>   color: pink !important;
> }
> /* stylelint-enable selector-max-id, declaration-no-important */
> ```

### What solution would you like to see?

I think we should explain this a bit more explicitly.

Maybe something like this:

> Or you can **scope configuration comments to** turn off individual rules:
> 
> ```css
> /* stylelint-disable selector-max-id, declaration-no-important */
> #id {
>   color: pink !important;
> }
> /* stylelint-enable selector-max-id, declaration-no-important */
> ```

(new text in bold)

**Comment by Mouvedia** [comment]
If it's an obvious documentation enhancement, with a small diff footprint, which you know is accurate, you can go ahead with the PR and skip filling an issue.
So Ill conjecture that you were unsure of the phrasing.

IMHO this kind of small improvements can be expedited.
i.e. you could even merge it yourself and skip the review

**Comment by romainmenke** [comment]
Our process is to open issues first and leave some room for discussion.
Even with trivial changes I find that there is value in this process :)

**Comment by ryo-manba** [comment]
I found the following explanation to be very clear.
see: https://stylelint.io/user-guide/options#fix

> - a scoped disable comment, e.g. /* stylelint-disable color-named */, any problems reported by the scoped rule will not be automatically fixed anywhere in the source
> - an unscoped disable comment, i.e. /* stylelint-disable */, the entirety of the source will not be automatically fixed for that rule


How about presenting it like this to make it clearer?

>  For example, you can either turn all the rules off **with an unscoped disable comment**:
> ```css
> /* stylelint-disable */
> a {}
> /* stylelint-enable */
> ```


> Or you can turn off individual rules **with a scoped disable comment**:
> ```css
> /* stylelint-disable selector-max-id, declaration-no-important */
> #id {
>   color: pink !important;
> }
> /* stylelint-enable selector-max-id, declaration-no-important */
> ```


(new text in bold)

**Comment by romainmenke** [comment]
Your suggestion seems good to me @ryo-manba!

**Comment by Mouvedia** [comment]
I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.
